content/docs/command-reference/remote/modify.md

```diff
@@ -343,7 +343,7 @@ By default, DVC authenticates using an `account_name` and its
 application. To use a custom authentication method, use the following parameters
 (listed in order of precedence):
 
-1. `connection_string` is used for authentication if given (all others params
+1. `connection_string` is used for authentication if given (all others parameters
```

Comment:
Is "params" confusing here? We call all these values (e.g. `connection_string`) configuration parameters or config params. Would "config params" be clearer?

Reply:
you're right, this is not really helping much. I'll rephrase it like so: `(all other params are ignored / should be removed from the config)`